### PR TITLE
Add Make in build-push-stolostron workflow

### PR DESCRIPTION
**Pull request by yiraeChristineKim**
`make` command is missed in` build-push-stolostron.yaml` file

### CODE REVIEW

**Review by dhaiducek — APPROVED** [review]
/hold if you want to make changes based on the comment

File: .github/workflows/build-push-stolostron.yaml [hunk 1]
```diff
@@ -28,5 +28,5 @@ jobs:
         REPOSITORY=${{env.REPOSITORY}} \
```
Comment (dhaiducek) [inline .github/workflows/build-push-stolostron.yaml:28]: Optional: I don't think it's necessary to define an `env` if it's only used here once.

**Review by mprahl — APPROVED** [review]
/hold in case you want to address the optional comment

File: .github/workflows/build-push-stolostron.yaml (hunk 1 above)
Comment (yiraeChristineKim) [reply .github/workflows/build-push-stolostron.yaml]: good idea

**Review by mprahl — APPROVED** [review]
/unhold

**Comment by mprahl** [comment]
@yiraeChristineKim does this need to be in the "master" branch as well for all other tags to work?

**Comment by yiraeChristineKim** [comment]
> @yiraeChristineKim does this need to be in the "master" branch as well for all other tags to work?
Dale and I don't use the master branch anymore. Dale created the v3.15 branch with rebasing v3.14

**Comment by dhaiducek** [comment]
@yiraeChristineKim I'll incorporate this commit into `release-3.15` in a rebase since there haven't been other changes there.

**Pull request merged**